ts/features/idpay/code/screens/IdPayCodeRenewScreen.tsx

```diff
@@ -0,0 +1,80 @@
+import {
+  Body,
+  H2,
+  IOStyles,
+  ListItemAction,
+  VSpacer
+} from "@pagopa/io-app-design-system";
+import * as React from "react";
+import { Alert, View } from "react-native";
+import { Link } from "../../../../components/core/typography/Link";
+import TopScreenComponent from "../../../../components/screens/TopScreenComponent";
+import I18n from "../../../../i18n";
+import { identificationRequest } from "../../../../store/actions/identification";
+import { useIODispatch } from "../../../../store/hooks";
+import { emptyContextualHelp } from "../../../../utils/emptyContextualHelp";
+
+export const IdPayCodeRenewScreen = () => {
+  const dispatch = useIODispatch();
+
+  const handleConfirm = () => {
+    dispatch(
+      identificationRequest(
+        false,
+        true,
+        undefined,
+        {
+          label: I18n.t("global.buttons.cancel"),
+          onCancel: () => undefined
+        },
+        {
+          onSuccess: () => null // TODO: handle success
+        }
+      )
+    );
+  };
+
+  return (
+    <TopScreenComponent
+      customGoBack={false}
+      dark={false}
+      goBack={true}
+      contextualHelp={emptyContextualHelp}
+    >
+      <View style={IOStyles.horizontalContentPadding}>
+        <H2>{I18n.t("idpay.code.renew.screen.header")}</H2>
+        <VSpacer size={16} />
+        <Body>{I18n.t("idpay.code.renew.screen.body")}</Body>
+        <Link>{I18n.t("idpay.code.renew.screen.link")}</Link>
+        <VSpacer size={16} />
+        <ListItemAction
+          label={I18n.t("idpay.code.renew.screen.generateCTA")}
+          onPress={() => customAlert(handleConfirm)}
+          icon="reload" // becomes `change` once new DS is released
```

Comment:
Can we add a task to address this later?

Reply:
addressed in [8b48f16](https://github.com/pagopa/io-app/pull/5011/commits/8b48f16bb174861716791d316eef352ed3dea781) 👍